models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -74,6 +73,9 @@ Users can install Graphviz for visualization of searched architectures (needed i
 
 ![Example of Searched Architecture](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_net_arch_search_segmentation_searched_arch_example_1.png)
 
+### System Memory Requirement
+[The provided searched architecture and trained checkpoints](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/large_files.yml) result from training conducted using 400GB of system RAM. Ample RAM is essential as we aim to cache all pre-processed data points in RAM to enhance the efficiency of model searching and training with [CacheDataset](https://docs.monai.io/en/stable/data.html#cachedataset). Should any errors occur during data pre-processing or caching in model search or training, kindly decrease the caching rate `cache_rate` in the configurations (e.g., [training](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/configs/train.yaml) and [multi-GPU training](https://github.com/dongyang0122/model-zoo/blob/dev/models/pancreas_ct_dints_segmentation/configs/multi_gpu_train.yaml)) within range (0, 1) to minimize the GPU RAM requirements.
```

Comment:
Hi @yiheng-wang-nv , we should not expose this Dong's Github repo here! the `large_files` is only used for development, end-users should not see it when download the bundle, the downloaded zip file already contains the model.pt in /models/ folder.

Thanks.